apps/web/src/pages/Pools/AssessmentPlanBuilderPage/components/OrganizeSection.tsx

```diff
@@ -201,10 +201,19 @@ const OrganizeSection = ({
       <p data-h2-margin="base(x1, 0)">
         {intl.formatMessage({
           defaultMessage:
-            "Use this section to define which assessments will be used as part of your assessment process. You can also change the order in which you plan to perform these evaluations. The only exceptions are the “Application screening” and the “Screening questions (at the time of application)” assessments which will always be the first and second steps in any assessment process.",
-          id: "0Arzgj",
+            "Use this section to define which assessments will be used as part of your assessment process. Make sure every essential skill is assessed at least once to complete your assessment plan.",
```

Comment:
See my note from this PR's description:

> [!IMPORTANT]  
> Some of the copy from this linked issue was changed by https://github.com/GCTC-NTGC/gc-digital-talent/issues/10195. I've done my best to incorporate it.

@Jerryescandon since you were the designer for both that issue and this one, could you have a look to confirm this is what is expected since they overlap?

Reply:
Yup, that's correct, the copy ticket ended up being worked un before this one but Matt's assumption is correct.
"...Make sure every essential skill is assessed" is the accurate copy moving forward.